pages/ksphere/konvoy/partner-solutions/jenkins/index.md

```diff
@@ -37,69 +43,69 @@ Your Jenkins instance will deploy.
     apiVersion: v1
     kind: ServiceAccount
     metadata:
-    name: jenkins
-    namespace: jenkins
+      name: jenkins
+      namespace: jenkins
     ---
     kind: Role
     apiVersion: rbac.authorization.k8s.io/v1beta1
     metadata:
-    name: jenkins
-    namespace: jenkins
+      name: jenkins
+      namespace: jenkins
     rules:
     - apiGroups: [""]
-    resources: ["pods"]
-    verbs: ["create","delete","get","list","patch","update","watch"]
+      resources: ["pods"]
+      verbs: ["create","delete","get","list","patch","update","watch"]
     - apiGroups: [""]
-    resources: ["pods/exec"]
-    verbs: ["create","delete","get","list","patch","update","watch"]
+      resources: ["pods/exec"]
+      verbs: ["create","delete","get","list","patch","update","watch"]
     - apiGroups: [""]
-    resources: ["pods/log"]
-    verbs: ["get","list","watch"]
+      resources: ["pods/log"]
+      verbs: ["get","list","watch"]
     - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get"]
+      resources: ["secrets"]
+      verbs: ["get"]
     ---
     apiVersion: rbac.authorization.k8s.io/v1beta1
     kind: RoleBinding
     metadata:
-    name: jenkins
-    namespace: jenkins
+      name: jenkins
+      namespace: jenkins
     roleRef:
-    apiGroup: rbac.authorization.k8s.io
-    kind: Role
-    name: jenkins
+      apiGroup: rbac.authorization.k8s.io
+      kind: Role
+      name: jenkins
     subjects:
     - kind: ServiceAccount
-    name: jenkins
+      name: jenkins
     EOF
     ```
 
 1. Create a `values.yaml` file that can be used to install Jenkins from the upstream helm repo.
 
     ```yaml
     master:
-    useSecurity: false
-    installPlugins: ["prometheus:2.0.6","kubernetes:1.18.2"]
-    csrf:
+      useSecurity: false
+      installPlugins: ["prometheus:2.0.6","kubernetes:1.18.2","workflow-job:2.33","workflow-aggregator:2.6","credentials-binding:1.19","git:3.11.0"] 
```

Comment:
for future reference:

these plugins were added to make sure this is identical to https://github.com/mesosphere/kubeaddons-enterprise/blob/dev/addons/jenkins/1.x/jenkins.yaml#L32-L37